playbooks/group_vars/prometheus/public.yml

```diff
@@ -25,6 +25,124 @@ prometheus_storage__tsdb__retention: 30d
 prometheus_web__console__templates: "{{ prometheus_install_dir }}/prometheus-{{ prometheus_version }}.{{ prometheus_platform_architecture }}/consoles"
 prometheus_web__console__libraries: "{{ prometheus_install_dir }}/prometheus-{{ prometheus_version }}.{{ prometheus_platform_architecture }}/console_libraries"
 
+PROMETHEUS_ALERT_CPU_THRESHHOLD: 50
+PROMETHEUS_ALERT_MEMORY_THRESHOLD: 90
+PROMETHEUS_ALERT_DISK_SPACE_THRESHOLD: 80
+
+PROMETHEUS_ALERT_HTTP_RESPONSE_TIME_THRESHOLD: 1
+PROMETHEUS_ALERT_HTTP_RESPONSE_FRONTEND_ERROR_THRESHOLD: 70
+
+PROMETHEUS_ALERT_HAPROXY_NODES_REGEX: '~".+"'
+PROMETHEUS_ALERT_HAPROXY_NODES_REGEX: 'haproxy.opencraft.hosting'
+
+prometheus_rules:
+  - name: NodeRules
+    rules:
+    - alert: instance_down
+      # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0/
+      expr: avg_over_time(up[5m]) < 0.91
+      for: '10s'
```

Comment:
Actually `prometheus_config_global_evaluation_interval` is set to `1m` so currently the rules are being evaluated once every 1 minute. I suspect even if `for` was set to a value lower that this, it would not get checked faster than that. Unfortunately the docs and googling did not turn up anything.

According to https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/ the default for `for` is `0s` so I'll change it to that.

Also the alertmanager receiver can wait before sending the first alert:
```
# How long to initially wait to send a notification for a group
# of alerts. Allows to wait for an inhibiting alert to arrive or collect
# more initial alerts for the same group. (Usually ~0s to few minutes.)
[ group_wait: <duration> | default = 30s ]
```
For pagerduty receiver I can change this to `0s` as well.

Reply:
@symbolist Hmm yeah I was pretty sure we'd have to update that evaluation interval if we wanted faster alerts. I guess we can have it be 30s which we can update in the secrets or something.